frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -80,11 +80,11 @@
             </v-col>
```

Comment:
On line 3 the min-width can be removed as it's now in AppDialog.

Reply:
Fixed.